app/views/case_logs/_log_filters.erb

```diff
@@ -16,6 +16,7 @@
                     <% years = {"2021": "2021/22", "2022": "2022/23"} %>
                     <%= render partial: "filters/checkbox_filter", locals: { f: f, options: years, label: "Collection year", category: "years" } %>
                     <%= render partial: "filters/checkbox_filter", locals: { f: f, options: status_filters, label: "Status", category: "status" } %>
+                    <%= render partial: "filters/checkbox_filter", locals: { f: f, options: {"all": "All", "yours": "Yours"}, label: "Logs", category: "user" } %>
```

Comment:
If you made this:

```
options: {"all": "", "yours": current_user.id}
```
Could you avoid the need for special case logic above?

Reply:
So we're using key as a param and value as label, it would need to be changed around. I feel like it'd be a bit weird to have a hash like this `{"In progress": "in_progress"}` but I guess possible